.github/workflows/website.yml

```diff
@@ -120,6 +145,7 @@ jobs:
         id: finder
       - name: Update Preview URLs PR Comment
         uses: marocchino/sticky-pull-request-comment@v2.2.0
+        continue-on-error: true
```

Comment:
I was hitting rate limits failures on this

Reply:
... and a bunch of other actions. should i check this in?

Reply:
you were seeing workflows fail due to hitting a rate limit during the `sticky-pull-request-comment` step?

Reply:
@olizilla Yeah, it was a several different steps, but that was one of them. I think it rate limiting was coming from github apis

Reply:
Let's grab that error and track down why it's happening. I don't want the website preview url step to introduce problems.

Reply:
I reverted this change. I don't think this PR introduced a root cause of why that happened. AFAICT it happens because we have enough workflow items that use `secrets.GITHUB_TOKEN` that run enough times per hour that we're getting rate limited by github's 1000req/hr default.